vignettes/rhino.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Setup shiny.telemetry in a Rhino application"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setup shiny.telemetry in a Rhino application}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The `{shiny.telemetry}` package can be used with any Shiny application and in this guide we will show how to use it with [{rhino} package](https://appsilon.github.io/rhino/).
+
+<div style="float: right;">
+<a href="https://appsilon.github.io/rhino/" target="_blank"><img src="https://appsilon.github.io/rhino/reference/figures/rhino.png" style="height: 140px;" /></a>
+<a href="https://appsilon.github.io/shiny.telemetry/"><img src="https://appsilon.github.io/shiny.telemetry/reference/figures/shiny-telemetry.png" style="height: 140px;" /></a>
+</div>
+
+`{rhino}` is a package that provides a framework to build high quality, enterprise-grade Shiny apps at speed.
+It is designed to help you build Shiny apps that are scalable, maintainable, and robust.
+
+Using `{shiny.telemtry}` with `{rhino}` is very simple and it requires only a few lines of code just as any other app.
+
+After initializing `{rhino}` with `rhino::init()` or using an existing `{rhino}` code base, you can start using `{shiny.telemetry}` to track events in your application by:
```

Comment:
1. Please add links to Rhino documentation that will guide users how they can initialize a Rhino app.
2. `shiny.telemetry` needs to be installed, please add a snippet with simple `rhino::pkg_install("shiny.telemetry")`

Reply:
Done.